src/components/avatar/avatar.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+/** @jsx jsx */
```

Comment:
What's this for? 🤔

Reply:
because the only JSX rendered by this component using the css prop, the built version drops all usages of react.createELement and replaces it with JSX

so we get warnings about unused 'react' import

Reply:
Here's an example of what `Text` looks like

```
import 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';
import requiredIf from 'react-required-if';
import { isNil } from 'lodash-es';
import { oneLine } from 'common-tags';
import { css, jsx } from '@emotion/core';
import vars from '@commercetools-uikit/custom-properties';
```

... more stuff

```jsx
var Text = function Text(_ref) {
  var intlMessage = _ref.intlMessage,
      children = _ref.children;
  return intlMessage ? jsx(FormattedMessage, intlMessage) : children;
};
```

Reply:
can remove this